db/schema.rb

```diff
@@ -235,6 +215,24 @@
     t.index ["dttp_id"], name: "index_dttp_users_on_dttp_id", unique: true
   end
 
+  create_table "funding_method_subjects", force: :cascade do |t|
+    t.bigint "funding_method_id"
+    t.bigint "allocation_subject_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["allocation_subject_id", "funding_method_id"], name: "index_funding_methods_subjects_on_ids", unique: true
+    t.index ["allocation_subject_id"], name: "index_funding_method_subjects_on_allocation_subject_id"
+    t.index ["funding_method_id"], name: "index_funding_method_subjects_on_funding_method_id"
+  end
+
+  create_table "funding_methods", force: :cascade do |t|
+    t.string "training_route", null: false
+    t.integer "amount", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.boolean "funding_type"
```

Comment:
looks wrong

Reply:
also should be non-nullable, but we need to do that after we migrate the data.